src/javascript/app/pages/user/account/settings.js

```diff
@@ -14,6 +15,24 @@ const Settings = (() => {
                 $('#change_password').setVisibility(1);
             }
 
+            // Disabling Authentication button for SVG accounts
+            const is_authenticated                 = State.getResponse('get_account_status.status').includes('authenticated');
```

Comment:
we already have `status` declared, so we can use that instead of `State.getResponse('get_account_status.status')`